### PR TITLE
AB#819 Adjust methods and types from the Drugly project

### PR DESCRIPTION
**Pull request by pmakowski23**

### CODE REVIEW

**Review by szymonbla — APPROVED** [review]
LGTM💪

**Review by mkidawa — APPROVED** [review]
LGTM

File: src/common/utils/cookies.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export function getCookie(name: string) {
```
Comment (JKrymarys) [inline src/common/utils/cookies.ts:1]: hmm I am not a fan of how this function looks like. It'd be great to have better variable names, and consistent style of conditions and loops (so not using one liners etc). 

However if it works and was used with success in Drugly then I guess that it's not a big isse

File: src/common/utils/localStorage.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function getItem(itemKey: string) {
+  const saved = window.localStorage.getItem(itemKey);
```
Comment (JKrymarys) [inline src/common/utils/localStorage.ts:2]: Minor thing, but maybe `savedValue` ?

**Review by JKrymarys — APPROVED** [review]
Two minor things. otherwise looks good 👍🏻

**Pull request merged**